.github/ISSUE_TEMPLATE/graduation.md

```diff
@@ -2,7 +2,8 @@
 - [ ] Have committers from at least two organizations.
 - [ ] Have achieved and maintained a Core Infrastructure Initiative Best Practices Badge.
 - [ ] Have completed an independent and third party security audit with results published of similar scope and quality as this example which includes all critical vulnerabilities and all critical vulnerabilities need to be addressed before graduation.
-- [ ] Explicitly define a project governance and committer process. This preferably is laid out in a GOVERNANCE.md file and references an OWNERS.md file showing the current and emeritus committers.
+- [ ] Explicitly define a project governance and committer process. The committer process should cover the full committer lifecycle including onboarding and offboarding or emeritus criteria. This preferably is laid out in a GOVERNANCE.md file and references an OWNERS.md file showing the current and emeritus committers.
```

Comment:
Not sure if we have glossary definitions for "committer" vs. "maintainer" somewhere?

I feel folks will need some clarify of how to evaluate this committer process criteria against their own contribution ladder (for example, many projects adopted the "approver + reviewer" system).

Reply:
Committer was the language used in the CNCF Foundation charter which I think is what lead to it being used in the various other docs. 

FWIW it is defined as:
"A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project." in the main graduation process readme:

https://github.com/cncf/toc/blob/c936591a9c17eac011b890387fcd4b8a653184b0/process/README.md?plain=1#L65

I do think that surfacing these terms in a more discoverable fashion would be good. I do think that most folk don't use that language these days, but changing it might have other ramifications unless the foundation charter is also updated :x